examples/units_and_coordinates/stereo_loc_solarmach.py

```diff
@@ -0,0 +1,53 @@
+"""
+===================================
+Getting the location of STEREO-A using solarmach
+===================================
+
+How to get the position of planetary bodies im the solar system using
+`astropy's solar system ephemeris <http://docs.astropy.org/en/stable/coordinates/solarsystem.html#solar-system-ephemerides>`__ information and sunpy.
+"""
+import datetime
+
+from solarmach import SolarMACH
+
+##############################################################################
+# These are necessary options
```

Comment:
Our examples try to expand on what is happening in the code, you should explain what these necessary options are.

Ideally link to the documentation that is relevant.